Symfony/CS/Tests/Tokenizer/TokenTest.php

```diff
@@ -240,4 +240,101 @@ public function testPropertiesOfNonArrayToken()
         $this->assertNull($token->getLine());
         $this->assertFalse($token->isArray());
     }
+
+    /**
+     * @dataProvider provideEquals
+     */
+    public function testEquals($token, $equals, $other, $caseSensitive = null)
+    {
+        if (!isset($caseSensitive)) {
```

Comment:
2 things here:

- we use ``null === $a`` to compare a variable to ``null``, not ``!isset($a)`` in the project.
- I don't see why you make it default to null and then call it without the argument instead of simply making ``$caseSensitive`` default to false